### PR TITLE
fix: passing the CIRCLE_TOKEN env var

### PR DESCRIPTION
**Pull request by michaeljherrmann**
Thanks for making this project! It was exactly what I was looking for, but I couldn't quite get it to work. I think there was a problem with how `CIRCLE_TOKEN` was being passed through and read. I think this is the fix, but please have a look!

### CODE REVIEW

File: src/jobs/notify.yml [hunk 1]
```diff
@@ -20,21 +20,21 @@ parameters:
...
       Channel name or ID will work.
       You may include a comma separated list of channels if you wish to post to multiple channels at once.
       Set the "SLACK_DEFAULT_CHANNEL" environment variable for the default channel.
-    default: SLACK_DEFAULT_CHANNEL
-    type: env_var_name
+    default: $SLACK_DEFAULT_CHANNEL
+    type: string
 
 steps:
-  - checkout
```
Comment (michaeljherrmann) [inline src/jobs/notify.yml:37]: Using this orb's job, I wouldn't expect it to be checking out my repo. Not sure if this was here for a specific or different use case?

File: src/commands/notify.yml [hunk 2]
```diff
@@ -17,18 +17,19 @@ parameters:
...
     description: >
       CircleCI token for fetching flaky tests via API.
+      If not specified, CIRCLE_TOKEN environment variable will be used.
       Please refer to this document how to generate it.
-      https://circleci.com/docs/managing-api-tokens
-    default: CIRCLE_TOKEN
-    type: env_var_name
+      https://circleci.com/docs/managing-api-tokens (requires a v2 token)
+    default: $CIRCLE_TOKEN
+    type: string
```
Comment (michaeljherrmann) [inline src/commands/notify.yml:24]: I'm following how the [circleci slack orb](https://github.com/CircleCI-Public/slack-orb/blob/3e43e753919b007d681cb2a16b37cabe5a8eda83/src/commands/notify.yml#L57) works. 
From what I understand we're supporting passing either the value or the name of an env variable.
It took me a bit to fully understand that so I probably would opt for a simpler interface, but I didn't want to change anything too dramatic here.

**Comment by michaeljherrmann** [comment]
@tadashi0713 do you think this is something you'll release? Otherwise I'll fork and release a fixed version, but I think it would be better to just push a fix for the one you already made.

**Review by tadashi0713 — APPROVED** [review]
Thanks, and sorry for late review. Looks good to me

**Comment by michaeljherrmann** [comment]
> Thanks, and sorry for late review. Looks good to me

Thanks! 🙂

**Pull request merged**